src/System.ComponentModel.EventBasedAsync/ref/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
     "System.ComponentModel.Primitives": "4.4.0-beta-24703-01",
+    "System.ComponentModel": "4.0.3.0",
```

Comment:
It needs to match the other versions, and specifically what's listed here:
https://github.com/dotnet/corefx/blob/master/dependencies.props#L28
Otherwise dependency validation will fail.